docker/wholegraph/install_wholegraph.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+git clone https://github.com/fmtlib/fmt.git  /opt/fmt
+cd /opt/fmt
+git checkout 9.1.0
+mkdir build && cd build
+cmake -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE ..
+make
+make install
+
+git clone https://github.com/gabime/spdlog.git  /opt/spdlog
+cd /opt/spdlog && mkdir build && cd build
+cmake .. && make -j
+cp libspdlog.a  /usr/lib/libspdlog.a
+export PYTHON=/usr/bin/python
+
+cd  /opt/rapids/
+git clone https://github.com/rapidsai/wholegraph.git -b refactoring
```

Comment:
The refactoring is for GraphStorm?

Reply:
@chang-l Can we use the master now?

Reply:
`refactoring` is their early dev/integration branch. Now you can safely move to their release branch `branch-23.08`.

Reply:
Fixed.